samples/msal-node-samples/ElectronSystemBrowserTestApp/README.md

```diff
@@ -99,39 +100,44 @@ npm start
 
 ### Custom URL Scheme
 
-To demonstrate best security practices, this Electron sample uses a custom URL scheme `msal4b0db8c2-9f26-4417-8bde-3f0e3656f8e0://auth` that will launch the app when the URL with that scheme is visited.
+To demonstrate best security practices, this Electron sample uses a custom URL scheme `msal{Your_Application/Client_Id}://auth` that will launch the app when the URL with that scheme is visited.
 
-Using this method, the desktop application starts the [Authorization code flow with PKCE flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) as normal by launching the system browser, authenticating the user, and redirecting with the authorization code in the browser.
+Using the [acquireTokenInteractive](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/0596334bde059484b6f37b1a2337af4e9655f9e9/lib/msal-node/src/client/PublicClientApplication.ts#L91) API, the desktop application starts the [Authorization code flow with PKCE flow](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) by launching and navigating the system browser to authorization code URL and listens for the authorization code response via loopback server. Once the code is received successfully, `acquireTokenInteractive` will load the assigned **successTemplate**.
```

Comment:
We should point to docs (TypeDocs are fine for now if more detailed docs don't exist yet) rather than the code.

Reply:
done